# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Hunter Wittenborn <hunter@hunterwittenborn.com>
# This file is distributed under the same license as the Celeste package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: Celeste 0.8.1\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-11-19 01:22+0000\n"
"PO-Revision-Date: 2024-03-01 09:00+0000\n"
"Last-Translator: H Tamás <hovanszki@gmail.com>\n"
"Language-Team: Hungarian <https://hosted.weblate.org/projects/celeste/"
"celeste/hu/>\n"
"Language: hu\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 5.5-dev\n"

#: src/about.rs:20
msgid "App icons by"
msgstr "App ikonokat készítette"

#: src/gtk_util.rs:20 src/gtk_util.rs:43 src/launch.rs:614 src/launch.rs:776
msgid "Ok"
msgstr "Ok"

#: src/launch.rs:99
msgid "Both '{}' and '{}' are more recent than at last sync."
msgstr "'{}' és '{}' frissebbek mint a legutolsó szinkelt."

#: src/launch.rs:158
msgid "Unable to create Celeste's config directory [{}]."
msgstr "A Celeste konfig mappájának létrehozása sikertelen [{}]."

#: src/launch.rs:170
msgid "Unable to create Celeste's database file [{}]."
msgstr "Sikertelen a Celesete adatbázis fájl létrehozása [{}]."

#: src/launch.rs:180
msgid "Unable to connect to database [{}]."
msgstr "Sikertelen adatbázishoz való csatlakozás [{}]."

#: src/launch.rs:188
msgid "Unable to run database migrations [{}]"
msgstr "Sikertelen adatbázis migráció futtatás [{}]"

#: src/launch.rs:288
msgid "Awaiting sync check..."
msgstr "Szinkelési ellenőrzésre várás..."

#: src/launch.rs:324
msgid "Sync Errors"
msgstr "Szinkronizációs Hibák"

#: src/launch.rs:339
msgid "File/Folder Exclusions"
msgstr "Fájl/Mappa Kivételek"

#: src/launch.rs:458
msgid "Stop syncing this directory"
msgstr "Ne szinkelje tovább ezt a mappát"

#: src/launch.rs:475
msgid "Are you sure you want to stop syncing '{}' to '{}'?"
msgstr "Biztos abban, hogy a '{}'-t nem szinkeli tovább '{}'-ba?"

#: src/launch.rs:484
msgid "This directory is currently being processed to no longer be synced."
msgstr "Ez a mappa jelenleg szinkelés megszüntetési folyamat alatt áll."

#: src/launch.rs:574
msgid "Directories"
msgstr "Mappák"

#. Get the local folder to sync with.
#: src/launch.rs:597
msgid "Local folder:"
msgstr "Helyi mappa:"

#: src/launch.rs:613 src/launch.rs:775 src/login/gdrive.rs:214
msgid "Cancel"
msgstr "Mégse"

#. Get the remote folder to sync with, and add it.
#. The entry completion code is largely inspired by https://github.com/gtk-rs/gtk4-rs/blob/master/examples/entry_completion/main.rs. I honestly have no clue what half the code for that is doing, I just know the current code is working well enough, and it can be fixed later if it breaks.
#: src/launch.rs:633
msgid "Remote folder:"
msgstr "Távoli mappa:"

#: src/launch.rs:820
msgid "The specified remote directory doesn't exist"
msgstr "A meghatározott távoli mappa nem létezik"

#: src/launch.rs:828
msgid "Failed to check if the specified remote directory exists"
msgstr "Nem sikerült ellenőrizni a távoli mappa létezését"

#: src/launch.rs:839
msgid "The specified directory pair is already being synced"
msgstr "A megadott mappa páros már alapból szinkelődik"

#: src/launch.rs:842
msgid "The specified local directory doesn't exist"
msgstr "A megadott helyi mappa nem létezik"

#: src/launch.rs:845
msgid "The specified local path isn't a directory"
msgstr "A megadott helyi elérési útvonal nem mappára mutat"

#: src/launch.rs:848
msgid "The specified local directory needs to be an absolute path"
msgstr "A megadott helyi mappa abszolút elérési úttal kell rendelkezzen"

#: src/launch.rs:876
msgid "Are you sure you want to delete this remote?"
msgstr "Biztosan ki akarja törölni ezt a távoli elérést?"

#: src/launch.rs:877
msgid "All the directories associated with this remote will also stop syncing."
msgstr "Minden ezzel a távoli eléréssel kapcsolatos mappa szinkelése leáll."

#: src/launch.rs:1342 src/launch.rs:2509
msgid "Files are synced."
msgstr "A fájlok szinkeltek."

#: src/launch.rs:1352
msgid "Syncing '{}'..."
msgstr "'{}' szinkelése..."

#: src/launch.rs:1373
msgid "Checking for changes..."
msgstr "Változások keresése..."

#. Add an error for reporting in the UI.
#: src/launch.rs:1380
msgid "Please resolve the reported syncing issues."
msgstr "Kérem oldja meg a jelentett szinkelési hibákat."

#: src/launch.rs:1407
msgid "{} errors found. "
msgstr "{} hibát találtunk. "

#: src/launch.rs:1421
msgid "Would you like to dismiss this error?"
msgstr "Szeretne eltekinteni ettől a hibától?"

#: src/launch.rs:1448
msgid "Failed to sync '{}' to '{}' on remote."
msgstr "'{}' szinkelése sikertelen ovlt '{}' o felé a távoli kapcsolatban."

#: src/launch.rs:1456
msgid "Failed to sync '{}' on remote to '{}'."
msgstr "'{}' sikertelenül szinkelt '{}' távolin."

#: src/launch.rs:1481
msgid "Unable to fetch data for '{}' from the remote."
msgstr "'{}' adatinak lekérdezése nem lehetséges a távoli kapcsolatról."

#: src/launch.rs:1490 src/launch.rs:1495 src/launch.rs:1503
msgid "File Update"
msgstr "Fájl Frissítés"

#: src/launch.rs:1490
msgid "Neither the local item or remote item exists anymore. This error will now be removed."
msgstr ""
"Sem a helyi sem a távoli elem nem áll rendelkezésre. Ez a hiba törlésre "
"kerül."

#: src/launch.rs:1495
msgid "Only the local item exists now, so it will be synced to the remote."
msgstr "Csak a helyi elem létezik, szinkelődni fog a távolira is."

#: src/launch.rs:1503
msgid "Only the remote item exists now, so it will be synced to the local machine."
msgstr "Csak a távoli elem létezik, ezért lokálisan is szinkelődni fog."

#: src/launch.rs:1513
msgid "Both the local item '{}' and remote item '{}' have been updated since the last sync."
msgstr ""
"Mind a helyi '{}' elem, mind a távoli '{}' elem frissítésre került a "
"legutóbbi szinkronizáció óta."

#: src/launch.rs:1515
msgid "Which item would you like to keep?"
msgstr "Melyik elemet kívánja meghagyni?"

#: src/launch.rs:1517
msgid "Local"
msgstr "Helyi"

#: src/launch.rs:1518
msgid "Remote"
msgstr "Távoli"

#: src/launch.rs:1567
msgid "1 error found."
msgstr "1 hibát találtunk."

#: src/launch.rs:1569
msgid "{} errors found."
msgstr "{} hibát találtunk."

#: src/launch.rs:1701
msgid "Checking '{}' for changes..."
msgstr "'{}' változásainak ellenőrzése..."

#: src/launch.rs:2117
msgid "Checking '{}' on remote for changes..."
msgstr "'{}' távoli változások ellenőrzése..."

#: src/launch.rs:2530
msgid "Finished sync checks with {} errors."
msgstr "{} hibával zárult a szikelés ellenőrzés."

#: src/launch.rs:2535
msgid "Finished sync checks."
msgstr "Szink ellenőrzés befejeződött."

#: src/login/gdrive.rs:211
msgid "Authenticating to {}..."
msgstr "Hitelesítése {}-be..."

#: src/login/gdrive.rs:212
msgid "Follow the link that opened in your browser, and come back once you've finished."
msgstr "Kövesse a böngészőben megnyíló linket, majd térjen vissza ha végzett."

#: src/login/gdrive.rs:239
msgid "There was an issue while running the webserver for authentication"
msgstr "Hiba lépett fel a webszerver hitelesítési futtatása során"

#: src/login/gdrive.rs:248
msgid "There was an issue authenticating to {}"
msgstr "Hiba lépett fel a {} hitelesítése során"

#: src/login/login_util.rs:15
msgid "Name"
msgstr "Név"

#: src/login/login_util.rs:27
msgid "Name already exists."
msgstr "Már létező név."

#: src/login/login_util.rs:29
msgid "Invalid name. Names must:\n"
"- Only contain numbers, letters, '_', '-', '.', and spaces\n"
"- Not start with '-' or a space\n"
"- Not end with a space"
msgstr ""
"Érvénytelen név. Tartalmaznia kell:\n"
"- Csak számokat, betűket, '_', '-', '.', és szóközt tartalmazhat\n"
"- Nem kezdődhet szóközzel vagy kötőjellel\n"
"- Nem végződhet szóközzel"

#: src/login/login_util.rs:43
msgid "Server URL"
msgstr "Szerver elérési cím"

#: src/login/login_util.rs:49
msgid "Invalid server URL ({})."
msgstr "Hibás szerver elérési cím ({})."

#: src/login/login_util.rs:58
msgid "Invalid server URL (no domain specified)."
msgstr "Hibás szerver elérési cím (domain nem került megadásra)."

#: src/login/login_util.rs:62
msgid "Invalid server URL (password was specified)."
msgstr "Hibás szerver elérési cím (megadott jelszó)."

#: src/login/login_util.rs:66
msgid "Invalid server URL(unknown server scheme {})."
msgstr "Hibás szerver elérési cím (ismeretlen szerver séma {})."

#: src/login/login_util.rs:78
msgid "Don't specify '{}' as part of the URL."
msgstr "Ne adja meg a '{}'-t a címben."

#: src/login/login_util.rs:91
msgid "Username"
msgstr "Felhasználónév"

#: src/login/login_util.rs:97
msgid "Password"
msgstr "Jelszó"

#: src/login/login_util.rs:104
msgid "2FA Code"
msgstr "2FA Kód"

#: src/login/login_util.rs:113
msgid "The provided 2FA code is invalid (should only contain digits)."
msgstr "A megadott 2FA kód helytelen (csak számokat tartalmazhat)."

#: src/login/login_util.rs:118
msgid "The provided 2FA code is invalid (should be 6 digits long)."
msgstr "A megadott 2FA kód helytelen (csak 6 karakter hosszú lehet)."

#: src/login/login_util.rs:142
msgid "Log in"
msgstr "Bejelentkezés"

#: src/login/mod.rs:77
msgid "Unable to connect to the server. Check your internet connection and try again."
msgstr ""
"A szerverhez való csatlakozás sikertelen. Kérem ellenőrizze az internet "
"kapcsolatát és próbálja újra."

#: src/login/mod.rs:80
msgid "A 2FA code is required to log in to this account. Provide one and try again."
msgstr ""
"2FA kód szükséges a fiókba való belépéshez. Kérem írja be a kódot és "
"próbálja újra."

#: src/login/mod.rs:83
msgid "Unable to authenticate to the server. Check your login credentials and try again."
msgstr ""
"Sikertelen hitelesítés a szerver felé. Ellenőrizze a bejelentkezési adatait, "
"és próbálja újra."

#: src/login/mod.rs:87
msgid "Unable to log in"
msgstr "Sikertelen bejelentkezés"

#. The dropdown for selecting the server type.
#: src/login/mod.rs:124
msgid "Server Type"
msgstr "Szerver Típus"

#: src/tray.rs:13
msgid "Awaiting sync checks..."
msgstr "Várakozás a szinkelési ellenőrzésre..."

#: src/tray.rs:60
msgid "Open"
msgstr "Megnyitás"

#: src/tray.rs:67
msgid "Close"
msgstr "Bezárás"

#: src/main.rs:188
msgid "Unknown Error"
msgstr "Ismeretlen Hiba"

#: src/main.rs:195
msgid "An unknown error has occurred while running. This is an internal issue with Celeste and should be reported.\n"
"\n"
"The following backtrace may help with debugging the issue - note that it may contain information such as login tokens/keys, so avoid posting the information publicly:"
msgstr ""
"A futás során ismetelen hiba lépett fel. Ez egy belső Celeste hiba, emiatt "
"jelentse be a hibát.\n"
"\n"
"A következő visszakövetés segíthet a hiba debugolásában - vegye figyelembe, "
"hogy az információ tartalmazhat bejelentkezési tokent/kulcsokat, ezért "
"tartózkodjon publikálásuktól:"
